packages/gatsby/src/services/run-page-queries.ts

```diff
@@ -50,18 +50,11 @@ export async function runPageQueries({
     !isCI()
   ) {
     cancelNotice = showExperimentNoticeAfterTimeout(
-      `queryOnDemand`,
-      reporter.stripIndent(`
-        Your local development experience is about to get better, faster, and stronger!
+      `Query On Demand`,
+      `https://gatsby.dev/query-on-demand-feedback`,
+      `which avoids running queries in development until you visit a page — so a lot less upfront work. Here's how to try it:
```

Comment:
```suggestion
      `which avoids running page queries in development until you visit a page — so a lot less upfront work. Here's how to try it:
```
Let's be accurate. It still isn't fully accurate because we run 404 and dev-404 queries due to how hardcoded they are in dev runtime, so maybe also add "avoids running MOST page queries" but not sure if this isn't going too far.

For the most part I don't want to give false impression that static queries are lazy (yet)